.portal-docs/docker-hub/README.aspnet.md

```diff
@@ -9,6 +9,10 @@
 
 # About
 
+> **NOTICE**: .NET Framework 3.5 images for Windows Server 2025 will no longer be published after May 13th.
```

Comment:
I'm not in favor of including this notice.  An announcement is sufficient IMO.  We have made more impactful changes in the past and never posted them in the readmes.  In a way doing this erodes the effort we have put into utilizing announcements.